### PR TITLE
Development

### LINKED ISSUE #28: Add expiration to json webtokens

### PR DESCRIPTION
**Pull request by NickRowlandson**
Resolves #28

### CODE REVIEW

File: app/client/app/components/attendance-list/attendance-list.component.ts [hunk 1]
```diff
@@ -274,11 +274,20 @@ export class AttendanceListComponent implements OnInit {
...
+        error.msg,
+        'info'
+      );
+    } else {
+      swal(
+        error.title,
+        error.msg,
+        'error'
+      );
+    }
```
Comment (ThomasLamb) [inline app/client/app/components/attendance-list/attendance-list.component.ts]: This `displayErrorAlert` method looks identical to `displayErrorAlert` methods in `attendance-report` and `case-notes`. Could this code be shared through a common inherited method or some other means?

File: app/client/app/components/client-status/client-status.component.ts [hunk 2]
```diff
@@ -611,6 +605,8 @@ export class ClientStatusComponent implements OnInit {
       title: 'Updating...'
     });
     swal.showLoading();
+    // var birthdate = new Date(this.clientEdit.birthdate);
+    // this.clientEdit.birthdate = moment(birthdate).format('MM/DD/YYYY');
```
Comment (ThomasLamb) [inline app/client/app/components/client-status/client-status.component.ts]: It's ok to delete unused code.

File: app/client/app/components/client-status/client-status.component.ts [hunk 3]
```diff
@@ -656,7 +652,9 @@ export class ClientStatusComponent implements OnInit {
           );
           this.clientView.email = this.currentClientEmail;
         } else if ((result as any).result === 'success') {
-          this.showStatusReport();
+          //this.showStatusReport();
```
Comment (ThomasLamb) [inline app/client/app/components/client-status/client-status.component.ts]: It's ok to delete unused code

File: app/client/app/components/attendance-list/attendance-list.component.ts (hunk 1 above)
Comment (NickRowlandson) [reply app/client/app/components/attendance-list/attendance-list.component.ts]: Absolutely. Will have that cleaned up for next pull request.

File: app/client/app/components/client-status/client-status.component.ts (hunk 3 above)
Comment (NickRowlandson) [reply app/client/app/components/client-status/client-status.component.ts]: Removed.

File: app/client/app/components/client-status/client-status.component.ts (hunk 2 above)
Comment (NickRowlandson) [reply app/client/app/components/client-status/client-status.component.ts]: Deleted.

**Pull request merged**